src/System.Collections.NonGeneric/tests/Performance/System.Collections.NonGeneric.Performance.Tests.csproj

```diff
@@ -20,10 +20,6 @@
     <ProjectReference Include="..\..\pkg\System.Collections.NonGeneric.pkgproj">
       <Name>System.Collections.NonGeneric</Name>
     </ProjectReference>
-    <!-- Do not remove this P2P reference since part of the implementation of NonGeneric has moved to Runtime.Extensions -->
-    <ProjectReference Include="..\..\..\System.Runtime.Extensions\pkg\System.Runtime.Extensions.pkgproj" />
```

Comment:
We might want to keep this one, since as the comment says, part of the implementation moved to System.Runtime.Extensions.